source/core_sntp_client.c

```diff
@@ -285,7 +285,6 @@ static SntpStatus_t sendSntpPacket( const UdpTransportInterface_t * pNetworkIntf
                                     size_t packetSize )
```

Comment:
Since this function is to be called on a single datagram, I don't think `size_t` is appropriate anymore for `packetSize`. Seems like `uint16_t` is more appropriate as detailed here: https://stackoverflow.com/questions/1098897/what-is-the-largest-safe-udp-packet-size-on-the-internet

Reply:
Good point, changed.